### PR TITLE
[Review] Request from 'schubi2' @ 'yast/autoyast-integration-test/review_170509_storage_ng'

### PR DESCRIPTION
**Pull request by schubi2**
Please review the following changes:
  * 4794b27 packaging
  * c5d40ed added environment for storage-ng

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -277,4 +277,6 @@ in order to initialize the socket communication.
 
 These AutoYaST integration tests are running on a SUSE Jenkins node:
 
-https://ci.suse.de/view/YaST/job/yast-autoyast-integration-test/
+https://ci.suse.de/view/YaST/job/yast-aytests-run-SLE12/
```
Comment (imobachgs) [inline README.md]: np: you could use a list:

```
* https://ci.suse.de/view/YaST/job/yast-aytests-run-SLE12/
* https://ci.suse.de/view/YaST/job/yast-aytests-run-CASP/
```

And do not forget the storage-ng tests.

File: lib/aytests/image_builder.rb [hunk 2]
```diff
@@ -18,9 +18,9 @@ class ImageBuilder
     IMAGE_BOX_NAME = "autoyast_vagrant_box_image_0.img".freeze
     SLEEP_TIME_AFTER_UPGRADE = 150
     SLEEP_TIME_AFTER_SHUTDOWN = 15
-    SSH_USER = "root".freeze
-    SSH_PASSWORD = "nots3cr3t".freeze
-    SSH_PORT = "22".freeze
+    SSH_USER = "root"
```
Comment (imobachgs) [inline lib/aytests/image_builder.rb]: why are you removing the `freeze` call?

**Review by imobachgs — CHANGES_REQUESTED** [review]
In general, it looks good. It's great that you managed to add support for libstorage-ng without touching too much code :)

File: lib/aytests/image_builder.rb (hunk 2 above)
Comment (schubi2) [inline lib/aytests/image_builder.rb]: due an issue in the ssh gem

Comment (imobachgs) [inline lib/aytests/image_builder.rb]: I see, the problem is here: https://github.com/busyloop/net-ssh-simple/blob/3bc03322d2cd69d97782c3c79045191f2d2428b1/lib/net/ssh/simple.rb#L635

**Review by imobachgs — APPROVED** [review]

**Pull request merged**